eclass/latex-package.eclass

```diff
@@ -206,10 +194,10 @@ latex-package_src_doinstall() {
 # relevant files that will be installed
 latex-package_src_compile() {
 	debug-print function $FUNCNAME $*
-	for i in `find \`pwd\` -maxdepth 1 -type f -name "*.ins"`
+	for i in `find $(pwd) -maxdepth 1 -type f -name "*.ins"`
 	do
 		einfo "Extracting from $i"
-		latex --halt-on-error --interaction=nonstopmode $i || die
+		latex --halt-on-error --interaction=nonstopmode ${i} || die
 	done
```

Comment:
crazy while read loop?

Reply:
I think we should put an example of that somewhere in devmanual.